java/src/jmri/IdTag.java

```diff
@@ -100,4 +121,26 @@
      */
     public void load(@Nonnull Element e);
 
+    /**
+     * Set an IdTag property
+     *
+     * @param key is the property to set
+     * @param value is the value to add to the set of properties
+     */
+    public void setProperty(String key,Object value);
```

Comment:
This is an @Override of the NamedBean method.  (See also the getProperty method)